pkg/xcp/xcp.go

```diff
@@ -211,6 +212,7 @@ func GetCIEnv(dir string) map[string]string {
 		values["SOLUBLE_METADATA_HOSTNAME"] = h
 	}
 
+	values["POLICY_METADATA_UPLOAD_TIME"] = time.Now().UTC().Format("2006-01-02T15:04:05.000Z")
```

Comment:
We usually record this on the server side instead of having the CLI provide it.

Also, this gets runs for every assessment upload, so if we really want to include this, and it's fine if we do, then it needs to be renamed.  Maybe simply UPLOAD_TIME.

There's a page on the internal wiki listing the upload variables and they're meaning.  Make sure to update that as well.